assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -227,7 +227,9 @@ function APIProvider({ children }) {
   );
 
   const getAllUsers = useCallback(() => {
-    return apiFetch({ path: addQueryArgs(users, { per_page: '-1' }) });
+    return apiFetch({
+      path: addQueryArgs(users, { per_page: '-1', who: 'authors' }),
```

Comment:
-1 still means it's loading all authors

Reply:
True, but isn't this the expected behavior based on the description in https://github.com/google/web-stories-wp/issues/4749?

Ah, okay, read through the comments again and saw that the part about loading only the first 100 authors first was in the comments. Thanks for pointing that out.